x-pack/legacy/plugins/ml/public/data_frame/pages/transform_management/components/transform_list/use_refresh_interval.ts

```diff
@@ -21,6 +22,7 @@ export const useRefreshInterval = (
   const { refresh } = useRefreshTransformList();
   useEffect(() => {
     let transformRefreshInterval: null | number = null;
+    let refreshIntervalSubscription: Subscription | undefined;
```

Comment:
In this file, `initAutoRefresh()` and `initAutoRefreshUpdate()` get called in any case and they are declared and called side by side. If you want, you could just remove those function declarations and just execute the code they contain in place. This would allow you to do `const refreshIntervalSubscription = timefilter...` instead of the `let` here.

Reply:
done. thanks!